aiqc/tests/__init__.py

```diff
@@ -378,8 +378,8 @@ def make_test_queue_keras_text_binary(repeat_count:int=1, fold_count:int=None):
 	)
 
 	encoderset.make_featurecoder(
-		sklearn_preprocess = PowerTransformer(method='yeo-johnson', copy=False)
-		, dtypes = ['object']
+		sklearn_preprocess = CountVectorizer(max_features = 200)
```

Comment:
were there any attributes of this class that need to be handled?

Reply:
Do you mean CountVectorizer?

Reply:
Yes. Not super important, but wondering if any of the options that CountVectorizer offers are problematic. For example, the `input` parameter should be forced to be `'content'`
https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html#sklearn.feature_extraction.text.CountVectorizer